### PR TITLE
Harvest some netfx assets

### PR DESCRIPTION
**Pull request by ViktorHofer**
Reopened from https://github.com/dotnet/corefx/pull/34384

Harvest System.Threading.AccessControl for netfx
Harvest S.C.Annotations for net461
Remove outdated netstandard configurations

### CODE REVIEW

File: src/System.ComponentModel.Annotations/ref/Configurations.props [hunk 1]
```diff
@@ -1,12 +1,7 @@
-﻿<Project>
+<Project>
   <PropertyGroup>
-    <PackageConfigurations>
-      netstandard;
```
Comment (ericstj) [inline src/System.ComponentModel.Annotations/ref/Configurations.props]: Have you chatted with @wtgodbe?  He’s doing some work which overlaps with this and you both may benefit from being on the same page.

**Comment by safern** [comment]
> BinaryFormatterTestData.cs(271,47): error CS0433: The type 'ValidationException' exists in both 'System.ComponentModel.Annotations, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' and 'System.ComponentModel.DataAnnotations, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\a\1\s\src\System.Runtime.Serialization.Formatters\tests\System.Runtime.Serialization.Formatters.Tests.csproj]

Seems like you might be missing a placeholder configuration on System.ComponentModel.Annotations?

**Comment by karelz** [comment]
@ViktorHofer no update for 6 weeks here. What is the plan?

**Comment by stephentoub** [comment]
@ViktorHofer, should this PR be closed?

**Comment by ViktorHofer** [comment]
No, this is currently blocked by https://github.com/dotnet/corefx/pull/38151. I should soon be able to make progress here.

**Comment by stephentoub** [comment]
@ViktorHofer, status on this?

**Comment by ViktorHofer** [comment]
Closing until I can make progress. Need to have a list somewhere with not yet finished but closed PRs...

**Comment by stephentoub** [comment]
> Need to have a list somewhere with not yet finished but closed PRs...

There should be an issue somewhere tracking whatever work this PR was trying to accomplish.

**Comment by ViktorHofer** [comment]
That sounds right. In infrastructure we have a lot of planned work items but probably not issues for everything. We should fix that :)

**Comment by ViktorHofer** [comment]
https://github.com/dotnet/corefx/issues/39086

**Pull request closed without merging**